src/js/utils/validator.ts

```diff
@@ -81,8 +81,8 @@ export function isDeepKeyCompliant(defaultObj: GenericObject, customObj: Generic
     const defaultKeys = Object.keys(defaultObj);
     return Object.keys(customObj).length >= defaultKeys.length &&
         defaultKeys.every(key => {
-            const defaultValue = defaultObj[key];
-            const customValue = customObj[key];
+            const defaultValue = defaultObj[key] as GenericObject;
```

Comment:
We're casting this to an object before checking if it's an object. This should be able to consume any value we can have for an object property, so this is a place implicit any should be fine.

Reply:
That makes sense. Knew there was a reason the cast was after this check before.